src/tests/banners/bannerSelection.test.ts

```diff
@@ -1,17 +1,18 @@
-import { selectBannerTest, _ } from './bannerSelection';
-import { cacheAsync as _cacheAsync } from '../../lib/cache';
-
-const cacheAsync = _cacheAsync;
-
-jest.mock('../../lib/cache', () => ({
```

Comment:
I've completely removed the use of mocks in the tests in favour of passing in dependencies to the `selectBannerTest` function. I find mocks incredibly hard to work with